CHANGELOG.next.toml

```diff
@@ -10,3 +10,9 @@
 # references = ["smithy-rs#920"]
 # meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "client | server | all"}
 # author = "rcoh"
+
+[[smithy-rs]]
+message = "Mark unconstrained structures with optional _or_ default members."
```

Comment:
This message is not useful to users at all. It makes no mention of the bug itself.

How about something like "Fix bug whereby nested server structure member shapes targeting simple shapes with `@default` produced Rust code that did not compile"?